app/frontend/src/features/profile/ProfileTagInput.tsx

```diff
@@ -120,6 +121,7 @@ export default function ProfileTagInput({
   options,
   label,
   id,
+  allowCsv,
```

Comment:
```suggestion
  allowCsv = true,
```
If it's true by default, you don't have to add it to each field (I can't see any reason we wouldn't want it most of the time).